src/lib/plugin.js

```diff
@@ -379,8 +379,8 @@ class FiveBellsLedger extends EventEmitter2 {
     if (!res || res.statusCode !== 200) { throwErr() }
 
     // note that the fivebells ledger API uses 'scale' instead of 'currency_scale'.
-    // the 'ilp_prefix' field is not mandatory
-    if (!res.body.currency_code || !res.body.scale) { throwErr() }
+    // fields like 'ilp_prefix' and 'currency_code' are not mandatory
+    if (!res.body.connectors || !res.body.scale) { throwErr() }
```

Comment:
ah, I think I know why. I wasn't using the `da-2.0` branch of five-bells-integration-test. Created a separate Dockerfile now which is rebased on da-2.0: https://github.com/michielbdejong/five-bells-integration-test/tree/mj-add-dockerfile-2.0